src/kv/idbstore.js

```diff
@@ -0,0 +1,39 @@
+// @ts-check
+
+const RELAXED = {durability: "relaxed"};
+
+const OBJECT_STORE = "chunks";
+
+/**
+ * @param {IDBDatabase} idb
+ * @return {IDBTransaction}
+ */
+export function readTransaction(idb) {
+  return idb.transaction(OBJECT_STORE, "readwrite");
```

Comment:
Two questions I guess:

(1) aren't const strings like `"chunks"` allocated statically anyway?
(2) do you want to make `"chunks"` and `"readwrite"` consistent?

Reply:
> (1) aren't const strings like "chunks" allocated statically anyway?

Not across the wasm boundary.

> (2) do you want to make "chunks" and "readwrite" consistent?

Yes

Reply:
You are right that string literals are allocated statically in V8. I misunderstood you.

I wanted `OBJECT_STORE` external so that it is not repeated

Reply:
But more importantly the read transaction should use `"readonly"`! Ooops!